tcms/testruns/forms.py

```diff
@@ -23,11 +23,26 @@ def populate(self, product_id):
         self.fields['build'].queryset = Build.objects.filter(product_id=product_id, is_active=True)
 
 
-class NewRunForm(BaseRunForm):
+class NewRunForm(forms.ModelForm):
+    class Meta:
+        model = TestRun
+        exclude = ('tag', 'cc')  # pylint: disable=modelform-uses-exclude
+
+    manager = UserField()
+    default_tester = UserField(required=False)
+
     case = forms.ModelMultipleChoiceField(
-        queryset=TestCase.objects.filter(case_status__id=2).all(),
+        queryset=TestCase.objects.filter(case_status=TestCaseStatus.get_confirmed()).all(),
```

Comment:
this causes failure b/c python tries to evaluate it before there is a database created. Many of the test commands use settings.test which points to a non-existing DB at this time.

Set it to objects.none() and move the initialization in .populate(). See rpc/api/forms/testrun.py::NewForm.assign_plan()